dashboard/src/scenes/report/view.js

```diff
@@ -174,7 +183,11 @@ const View = () => {
                 <ButtonCustom color="link" className="noprint" title="Suivant" disabled={reportIndex === 0} onClick={onNextReport} />
               </div>
             </div>
-            <div>{`Compte rendu du ${toFrenchDate(report.date)} de l'équipe ${currentTeam.name}`}</div>
+            <div>
+              {`Compte rendu de l'équipe ${currentTeam.name}`}
```

Comment:
J'ai vu que dans les autres endroits tu avais traité le cas d'absence de currentTeam, je ne sais pas si c'est voulu (autrement dit tu peux ignorer ma suggestion) : 

```suggestion
              {`Compte rendu de l'équipe ${currentTeam?.name || ""}`}
```